ci/travis_script_python.sh

```diff
@@ -92,7 +92,10 @@ python_version_tests() {
   conda install -y -q nomkl
 
   # Expensive dependencies install from Continuum package repo
-  conda install -y -q pip numpy pandas cython
+  conda install -y -q pip numpy pandas cython flake8
+
+  # Fail fast on style checks
+  flake8 pyarrow
```

Comment:
I took a look at this. I figured out how to get flake8 to run on Cython files, and I don't think that can be run within pytest-flake8, so probably simplest to leave this as is (now there's an extra call to check Cython files with some flake8 checks disabled)